src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -235,7 +241,12 @@ def add_realpath(path):
         realpaths.add(realpath)
 
     cp = configparser.SafeConfigParser()
-    cp.readfp(StringIO(self.DEFAULT_COVERAGE_CONFIG))
+    coverage_config_file = self.get_options().coverage_config_file
+    if coverage_config_file is not None:
+      with io.open(coverage_config_file, 'rt') as fp:
```

Comment:
Trapping errors and re-raising via a [TaskError](https://github.com/pantsbuild/pants/blob/master/src/python/pants/base/exceptions.py#L11-L32) subclass would be best. Subclassing is useful in particular for tests and raising `TaskError` is something understood by the outer presentation layers.